### PR TITLE
updated v0.13.5

### PR DESCRIPTION
**Pull request by flaviomartins**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes: https://github.com/conda-forge/duckdb-engine-feedstock/pull/59
Closes #58 
Closes #61 

@conda-forge-admin please re-render

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {% set name = "duckdb-engine" %}
 {% set package_name = name | replace("-", "_") %}
-{% set version = "0.13.2" %}
+{% set version = "0.13.5" %}
+{% set python_min = python_min|default("3.8") %}
```
Comment (pavelzw) [inline recipe/meta.yaml:4]: why do we need this?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,22 +18,24 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python {{ python_min }}.*
     - pip
     - poetry
```
Comment (pavelzw) [inline recipe/meta.yaml:23]:
```suggestion
    - poetry-core >=1.0.0
```

https://github.com/Mause/duckdb_engine/blob/c67c2e0e4f0d1b805c0917ef8372697541965eff/pyproject.toml#L64

File: recipe/meta.yaml (hunk 1 above)
Comment (flaviomartins) [reply recipe/meta.yaml:4]: Seems to be suggested by the conda-forge linter to pickup python-min from conda-forge-pinning (currently 3.9). Reference: https://github.com/conda-forge/cfep/blob/main/cfep-25.md

Comment (flaviomartins) [reply recipe/meta.yaml:4]: I'm reading now that we should override python_min in the case the minimum supported python is a larger version... Do you mean we don't need set python_min here and just let the global minimum python version be 3.9?

Comment (pavelzw) [reply recipe/meta.yaml:4]: yeah, i would just do `>= 3.9` everywhere as `<3.9` is EOL anyway

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (pavelzw) [reply recipe/meta.yaml]: [`3285584` (#60)](https://github.com/conda-forge/duckdb-engine-feedstock/pull/60/commits/32855843677c4f563ed3e1a39f627152c03c63d1)

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (pavelzw) [reply recipe/meta.yaml]: sorry for the confusion, the `python_min` linting rule apparently is new and i didn't know it yet

**Pull request merged**